helm/promtail/templates/service.yaml

```diff
@@ -1,15 +1,16 @@
+{{- if not .Values.promtail.serviceMonitor.enabled }}
```

Comment:
I'd rather use a new, giantswarm-specific value (like `.Values.giantswarm.labelmonitoring.enabled`).
- avoids relying on a subchart's value, which could cause visibility issues
- better clarity on what the value does

Reply:
Done